lib/integration/queryManager.js

```diff
@@ -0,0 +1,34 @@
+import { logMessage } from "../logger";
+import { dbConnector } from "./dbConnector";
+
+export const executeQuery = async (sql, params) => {
+  const client = await dbConnector();
```

Comment:
pass in the client instead of instantiating it from here. Otherwise everytime I call this executeQuery function a new connection is established and returned.